gradlew

```diff
@@ -116,7 +116,7 @@ case "$( uname )" in                #(
   NONSTOP* )        nonstop=true ;;
 esac
 
-CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
+CLASSPATH="\\\"\\\""
```

Comment:
Does this change comes from a new version of the Gradle wrapper?  It seems strange that the `CLASSPATH` value is set to `\"\"`

Reply:
Yes, it seems to come from gradle upgrade.